lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
Checking http://lisperator.net/uglifyjs/ast, I think 22 makes more sense as a result, `AST_Accessor` is a child of `AST_Lambda`, and has no name. I am committing your implementation and adjusting the test,